shared/AppInsightsCore/src/JavaScriptSDK/BaseCore.ts

```diff
@@ -43,6 +44,7 @@ export class BaseCore implements IAppInsightsCore {
         _this.isInitialized = () => _isInitialized;
         _this._setInit = (value: boolean) => { _isInitialized = value; }
         _this._eventQueue = [];
+        _this._UUID_Regex='^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$';
```

Comment:
As the backend is most likely written in .Net, I think we should support all valid forms (except maybe the forms wrapped in { } brackets.

But I would expect that the hyphens should be optional, so I would think we should support these 2

Specifier | Description | Format
-- | -- | --
N | 32 digits | 00000000000000000000000000000000
D | 32 digits separated by hyphens | 00000000-0000-0000-0000-000000000000